helm/cloud-provider-azure/README.md

```diff
@@ -139,13 +139,48 @@ The following configuration is made available for advanced users. There are no d
 
 # Maintaining the Repo
 
-Whenever changes have been made to the `cloud-provider-azure` helm chart, a new version of the chart should be released. First, pick an appropriate next, higher version and update the `version` property in `helm/cloud-provider-azure/Chart.yaml`. Then, package the entire set of changes to the chart into a new repo version:
+Whenever changes have been made to the `cloud-provider-azure` Helm chart, a new version of the chart should be released. First, pick an appropriate next, higher version and update the `version` property in `helm/cloud-provider-azure/Chart.yaml`. Then, package the entire set of changes to the chart into a new repo version:
 
 From the git root:
 
 ```bash
 $ make update-helm
-Successfully packaged chart and saved it to: helm/repo/cloud-provider-azure-1.23.8.tgz
+Successfully packaged chart and saved it to: helm/repo/cloud-provider-azure-1.23.11.tgz
 ```
 
-Changes to the chart should *always* include a new version, and then an update to the helm repo as described above.
+Changes to the chart should *always* include a new version, and then an update to the Helm repo as described above.
+
+If changes to the chart don't include an update to the Helm repo as described above, then the PR CI job will fail, preventing a merge to `master`.
+
+To check if your Helm chart changes will pass CI, you can manually run the verification script:
+
+```bash
+$ hack/verify-helm-repo.sh
+++ dirname hack/verify-helm-repo.sh
```

Comment:
is it possible to move the script into a file?

Reply:
@nilo19 this is actually not a script. Just run the command `hack/verify-helm-repo.sh` and its output.

Reply:
@jackfrancis I think you ran the old script and could you please rerun with the latest one? I just updated it yesterday.
https://github.com/kubernetes-sigs/cloud-provider-azure/pull/1656#event-6556909503

Reply:
updated output rebasing on top of your script change, thanks!